cglims/api.py

```diff
@@ -39,8 +39,12 @@ def pipeline(self):
 
         Returns (str): 'mip' or 'mwgs'
         """
-        if self.lims.udf.get('tissue_type') != 'tumour':
-            return 'mip' if self.apptag.is_human else 'mwgs'
+        if self.lims.get('tissue_type') != 'tumour':
+            if self.apptag.is_human:
+                return 'mip'
+
+        if self.apptag.is_microbial:
```

Comment:
hoh. I could change that, yes. I usually like how if statements are stand-alone when they are used for flow control. You know that each if ends in a `return` and I find it convoluted to add `elif` there.
But if it makes it clearer, of course!